libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/CompiledGraphQL.kt

```diff
@@ -375,37 +375,37 @@ fun resolveVariables(value: Any?, variables: Executable.Variables): Any? {
   }
 }
 
-@Deprecated("Introspection types are now generated like other types. Use the generated class instead.", level = DeprecationLevel.ERROR)
+@Deprecated("Introspection types are now generated like other types. Use the generated class instead.")
```

Comment:
Why not ERROR any more?

Reply:
Woops sorry I should have commented yes! This is because these are still used in the code generated by 3.x. With the error level it wouldn't compile. With this change, it compiles with a warning (I had to use `allWarningsAsErrors(false)` because of that - which is a shame but can be removed later when we use 4.x).

Reply:
See below, this should work if using the same runtime version as the version of the codegen

Reply:
Yes looks like it's working 🎉 (seems to be confusing IJ though, but maybe it's a fluke)